src/Features/CSharp/Portable/ReplacePropertyWithMethods/CSharpReplacePropertyWithMethodsService.cs

```diff
@@ -110,13 +108,13 @@ private static SyntaxNode GetSetMethod(
         IMethodSymbol setMethod,
         string desiredSetMethodName,
         ExpressionBodyPreference expressionBodyPreference,
+        bool copyLeadingTrivia,
         CancellationToken cancellationToken)
     {
         var methodDeclaration = GetSetMethodWorker();
 
-        // The analyzer doesn't report diagnostics when the trivia contains preprocessor directives, so it's safe
-        // to copy the complete leading trivia to both generated methods.
```

Comment:
Did this comment used to be true? It seems like the code was written with consideration to pp directives, so I'm curious where it went wrong.

Reply:
yeah.  that comment looks to be a big fat lie.  it might have been true way in the past.  :)